cmake/Modules/FindCodec2.cmake

```diff
@@ -17,17 +17,16 @@ INCLUDE(FindPackageHandleStandardArgs)
 pkg_check_modules(LIBCODEC2_PKG QUIET codec2)
 
 find_path(LIBCODEC2_INCLUDE_DIR NAMES codec2.h
+  HINTS ${LIBCODEC2_PKG_INCLUDE_DIRS}
```

Comment:
I guees this might be a good change in other cmake modules as well. Only hardcoded paths should go into `PATHS`.

Reply:
Yes, although practically it probably doesn't make too much of a difference. For anyone curious, the practical difference between HINTS and PATHS is that CMake will search standard system paths before PATHS but after HINTS.